src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -23,7 +23,7 @@
   }
 
   .additional-resources {
-    @include media-maxwidth($medium-large-screen) {
+    @include media-maxwidth($medium-large-screen - 1) {
```

Comment:
At the risk of barking up the wrong tree, this is a good example of why designing for mobile first is important. Adding additional functionality for larger screens would have solved this problem; these breakpoints are for minimum widths, not maximums.

:man_shrugging: